experiment/oss_fuzz_checkout.py

```diff
@@ -296,6 +296,47 @@ def is_image_cached(project_name: str, sanitizer: str) -> bool:
     return False
 
 
+def rewrite_project_to_cached_project_cloud_ccache(generated_project) -> None:
```

Comment:
nit: is this ccache specific? Or just chronos? 

The exact same steps will work for the autogenerated replay build scripts right?

Reply:
yes, changed the name